### ISSUE TITLE
Issue with Reproducing Quantitative Results

### ISSUE THREAD

**Issue by sysuhuangwenmin** [yusufdalva/VecGAN#8]
@yusuf.dalva.dev@gmail.com I am unable to reproduce the FID metric results in Table 3 of the paper using the released pre-trained model. Could you provide more details on how the FID was calculated? Specifically:

1、Were the generated images produced using reference guidance or noise guidance? For each test image, were 5 edited images generated or just 1 edited image?
2、Were the real images used for FID calculation from the training set or the test set, and were they at a resolution of 1024 or 256?
